From: Julian Ospald <hasufell@gentoo.org>
Date: Thu Aug  2 22:12:00 UTC 2012
Subject: underlinking

add -lpthread to cockatrice amd servatrice LIBS to support linkers that
don't permit underlinking

--- cockatrice/CMakeLists.txt
+++ cockatrice/CMakeLists.txt
@@ -219,7 +219,7 @@
 INCLUDE_DIRECTORIES(${QT_MOBILITY_MULTIMEDIAKIT_INCLUDE_DIR})
 
 ADD_EXECUTABLE(cockatrice WIN32 MACOSX_BUNDLE ${cockatrice_SOURCES} ${cockatrice_QM} ${cockatrice_RESOURCES_RCC} ${cockatrice_HEADERS_MOC})
-TARGET_LINK_LIBRARIES(cockatrice cockatrice_common ${QT_LIBRARIES} ${QT_MOBILITY_MULTIMEDIAKIT_LIBRARY})
+TARGET_LINK_LIBRARIES(cockatrice cockatrice_common ${QT_LIBRARIES} ${QT_MOBILITY_MULTIMEDIAKIT_LIBRARY} -lpthread)
 
 INSTALL(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/cockatrice DESTINATION bin)
 if (NOT WIN32 AND NOT APPLE)
--- servatrice/CMakeLists.txt
+++ servatrice/CMakeLists.txt
@@ -46,7 +46,7 @@
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
 
 ADD_EXECUTABLE(servatrice ${servatrice_SOURCES} ${servatrice_HEADERS_MOC})
-TARGET_LINK_LIBRARIES(servatrice cockatrice_common ${QT_LIBRARIES} ${LIBGCRYPT_LIBRARY})
+TARGET_LINK_LIBRARIES(servatrice cockatrice_common ${QT_LIBRARIES} ${LIBGCRYPT_LIBRARY} -lpthread)
 
 #add_custom_target(versionheader ALL DEPENDS version_header)
 add_custom_command(
